File: src/main/java/org/manapart/endersort/ItemGroupIF.java
package org.manapart.endersort;

import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;

public class ItemGroupIF extends ItemGroup {
    public static final ItemGroupIF instance = new ItemGroupIF(ItemGroup.getGroupCountSafe(), "endersort");

    private ItemGroupIF(int index, String label) {
        super(index, label);
    }

    @Override
    public ItemStack makeIcon() {
        return new ItemStack(Endersort.endersortIcon);
    }

}

File: src/main/java/org/manapart/endersort/ItemGroupIF.kt
package org.manapart.endersort

import net.minecraft.world.item.CreativeModeTab
import net.minecraft.world.item.ItemStack

class ItemGroupIF private constructor(index: Int, label: String) : CreativeModeTab(index, label) {
    override fun makeIcon() = ItemStack(Endersort.icon)

    companion object {
        val instance = ItemGroupIF(getGroupCountSafe(), "endersort")
    }
}